tests/integ/sagemaker/jumpstart/private_hub/model/test_jumpstart_private_hub_model.py

```diff
@@ -0,0 +1,174 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import time
+
+import pytest
+from sagemaker.enums import EndpointType
+from sagemaker.jumpstart.hub.hub import Hub
+from sagemaker.jumpstart.hub.utils import generate_hub_arn_for_init_kwargs
+from sagemaker.predictor import retrieve_default
+
+from sagemaker.jumpstart.constants import JUMPSTART_LOGGER
+
+import tests.integ
+
+from sagemaker.jumpstart.model import JumpStartModel
+from tests.integ.sagemaker.jumpstart.constants import (
+    ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME,
+    ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID,
+    JUMPSTART_TAG,
+)
+from tests.integ.sagemaker.jumpstart.utils import (
+    get_public_hub_model_arn,
+    get_sm_session,
+)
+
+MAX_INIT_TIME_SECONDS = 5
+
+TEST_MODEL_IDS = {
+    "catboost-classification-model",
+    "huggingface-txt2img-conflictx-complex-lineart",
+    "meta-textgeneration-llama-2-7b",
+    "meta-textgeneration-llama-3-2-1b",
+    "catboost-regression-model",
+}
```

Comment:
I think the integ test runs in PDX. Double check these are available in PDX region.

Reply:
these should be, I have chosen this models from the existing Jumpstart hub integ tests